app/src/main/java/com/seoulfitu/android/data/local/entity/SportsServiceScrapEntity.kt

```diff
@@ -30,4 +30,5 @@ data class SportsServiceScrapEntity(
     val operatingEndTime: String,
     val cancellationCriteria: String,
     val timeLeftForCancellation: Int,
+    val type:SportsServiceType
```

Comment:
Entity에는 class 형으로 저장이 될 수 없어서 그래서 여기에서 에러가 뜨는것 같아요!🥹
아마 Domain은 해당 클래스 타입으로 해놓고 Mapper에서 Entity로 바꿔줄때 String으로 바꿔줘야 할것 같슴다!
```suggestion
    val type: String
```

Reply:
제가 할 때도 그 오류가 났었는데요 이거 아마 뽀또씌 폰에 깔려있던 앱의 데이터베이스 타입이 현재 브랜치랑 안맞아서 저런 오류가 났었던 걸로 기억합니다!!! 😭😭😭 그래서 앱 지우고 다시 깔면 정상적으로 실행이 될 것 같습니다!!! 😍😍😍😍 한번 다시 확인 부탁드립니댱 히히히

Reply:
꺄올!!!! 됐어요!!!! 그래서 그랬구나!!! ㅋㅋㅋㅋㅋ
친절한 설명 감사합니당😘 내 가상 기기 문제였던 걸루><